pygmo/docstrings.cpp

```diff
@@ -1938,4 +1938,155 @@ Runs the fast non dominated sorting algorithm on the input *points*
 )";
 }
 
+std::string hv_init1_docstring()
+{
+    return R"(__init__(pop)
+
+Constructor from population
+
+Args:
+    pop (``pygmo.population``): a pygmo.population
```

Comment:
I'd put a link to the pygmo docs for population here in the type specification (without the backticks as sphinx already takes care of formatting during link generation). Description might be "the input population" instead?